src/javascript/binary/pages/trade/price.js

```diff
@@ -37,7 +37,7 @@ var Price = (function () {
             prediction = document.getElementById('prediction');
 
         if (payout && payout.value) {
-            proposal['amount_val'] = parseFloat(payout.value);
+            proposal['amount'] = parseFloat(payout.value);
```

Comment:
these all have been fixed and is live in master. merge latest master as there will be conflicts